src/modules/billing/services/batch-service.js

```diff
@@ -55,6 +56,34 @@ const getMostRecentLiveBatchByRegion = async regionId => {
   return batch ? mappers.batch.dbToModel(batch) : null;
 };
 
+const getDuplicateSentBatch = async (regionId, batchType, toFinancialYearEnding, isSummer) => {
+  const batches = await newRepos.billingBatches.findSentBatchesForRegion(regionId);
+  const batch = batches.find(b => b.batchType === batchType &&
+      b.toFinancialYearEnding === toFinancialYearEnding &&
+      b.isSummer === isSummer);
+
+  return batch ? mappers.batch.dbToModel(batch) : null;
+};
+
+/**
+ * Checks if there is:
+ *  - an existing batch in flight for the region
+ *  - a given batch for the same type, region, year and season (excluding supplementary)
+ * @param {String} regionId
+ * @param {String} batchType
+ * @param {Number} toFinancialYearEnding
+ * @param {Boolean} isSummer
+ * @return {Batch|null} if batch exists
+ */
+const getExistingOrDuplicateSentBatch = async (regionId, batchType, toFinancialYearEnding, isSummer) => {
```

Comment:
At the moment it seems we are doing 2 separate queries and then some additional filtering to find the duplicate/sent batches.

Could this be simplified by:
* A query finds all batches in region
* Putting them through a single `find()` which looks for either an open batch in the same region, or a sent batch of the same type/region/financial year/season (if the requested new batch is annual/two_part_tariff)

I think we may then need fewer repo methods too